ansible_runner/config/runner.py

```diff
@@ -243,6 +243,13 @@ def prepare_env(self):
         self.env["RUNNER_OMIT_EVENTS"] = str(self.omit_event_data)
         self.env["RUNNER_ONLY_FAILED_EVENTS"] = str(self.only_failed_event_data)
 
+        original_stdout_callback = os.getenv('ANSIBLE_STDOUT_CALLBACK')
+        if original_stdout_callback:
+            self.env['ORIGINAL_STDOUT_CALLBACK'] = original_stdout_callback
+
+        self.env['ANSIBLE_STDOUT_CALLBACK'] = 'awx_display'
+        self.env["ANSIBLE_LOAD_CALLBACK_PLUGINS"] = '1'
```

Comment:
I don't like that we ever modified `self.env` in the runner module to begin with. It's an oddball. Most of the environment is prepared in the config class, and this is an exception that I don't feel like we ever needed.